config/crd/bases/consul.hashicorp.com_proxydefaults.yaml

```diff
@@ -80,6 +80,13 @@ spec:
                     description: Mode is the mode that should be used for the upstream connection. One of none, local, or remote.
                     type: string
                 type: object
+              transparentProxy:
+                description: TransparentProxy controls configuration specific to proxies in transparent mode.
+                properties:
+                  outboundListenerPort:
+                    description: The port of the listener where outbound application traffic is being redirected to.
```

Comment:
Do you think we should add something like "this would not typically be set via the CRD, but this field exists to match the config-entry", or is it not worth it with the validation error?

Reply:
When a user does try to explicitly set this field, a validation error is returned from the webhook 😄 
The error message returned by the webhook is: Invalid value: v1alpha1.TransparentProxy{OutboundListenerPort:<value>}: use the annotation `consul.hashicorp.com/transparent-proxy-outbound-listener-port` to configure the Outbound Listener Port

Reply:
Yeah I had noticed that, and that should be enough :)